fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/util/BulkDataUtil.java

```diff
@@ -284,4 +287,37 @@ public static String checkAndValidateJob(Parameters parameters) throws FHIROpera
 
         throw new FHIROperationException("no job identifier is passed");
     }
+
+
+    public static String encryptBatchJobId(String strToEncrypt, SecretKeySpec key) {
```

Comment:
worthy of a comment

Reply:
let me add, I thought it's straight forward, so I didn't.

Reply:
done